clang/test/Sema/aix-attr-tls_model.c

```diff
@@ -6,6 +6,6 @@
 #endif
 
 static __thread int y __attribute((tls_model("global-dynamic"))); // no-warning
-static __thread int y __attribute((tls_model("local-dynamic"))); // expected-error {{TLS model 'local-dynamic' is not yet supported on AIX}}
+static __thread int y __attribute((tls_model("local-dynamic"))); // expected-no-diagnostics
```

Comment:
Thank you for asking. I did try to change it to "no-warning", however I got error message from llvm-lit complaining that since there is no diagnostic message anymore, we should "expected-no-diagnostics". So I did this change to make llvm-lit happy. If there is other formal/better setting, I will be happy to make the change.

Reply:
The particular error for setting the check to "no-warning" is:

clang -cc1 -internal-isystem $llvm-public/build/lib/clang/18/include -nostdsysteminc -triple powerpc64-unknown-aix -target-cpu pwr8 -verify -fsyntax-only $llvm-project/clang/test/Sema/aix-attr-tls_model.c
error: no expected directives found: consider use of 'expected-no-diagnostics'
1 error generated.